sdk/cosmos/azure-cosmos-benchmark/ctl/linkedin/run_benchmark.sh

```diff
@@ -0,0 +1,109 @@
+#!/bin/bash
+
+############################################################################
+## The parameters supported
+##  - ctl_endpoint (required)
+##  - ctl_key (required)
+##  - ctl_database (optional: default testdb)
+##  - ctl_collection (optional: default testCol)
+##  - ctl_throughput (optional: default 100,000)
+##  - ctl_concurrency (optional: default 50)
+##  - ctl_consistency_level (optional: default Session)
+##  - ctl_number_of_precreated_documents (optional: default 100,000)
+##  - ctl_number_of_operations (optional: default 1,000,000)
+##  - ctl_max_running_time_duration (optional: default 10 minutes)
+##  - ctl_printing_interval (optional: default 30 seconds)
+
+service_endpoint=$ctl_endpoint
+master_key=$ctl_key
+operation=LinkedInCtlWorkload
+number_Of_collection=1
+connection_mode=Direct
+protocol=Tcp
+
+if [ -z "$ctl_database" ]
+then
+database_name="testdb"
+else
+database_name=$ctl_database
+fi
+
+if [ -z "$ctl_collection" ]
+then
+collection_name="testCol"
+else
+collection_name=$ctl_collection
+fi
+
+if [ -z "$ctl_throughput" ]
+then
+throughput=100000
+else
+throughput=$ctl_throughput
+fi
+
+if [ -z "$ctl_concurrency" ]
+then
+concurrency=50
+else
+concurrency=$ctl_concurrency
+fi
+
+if [ -z "$ctl_consistency_level" ]
+then
+consistency_level=Session
+else
+consistency_level=$ctl_consistency_level
+fi
+
+if [ -z "$ctl_number_of_precreated_documents" ]
+then
+number_of_precreated_documents=100000
+else
+number_of_precreated_documents=$ctl_number_of_precreated_documents
+fi
+
+if [ -z "$ctl_number_of_operations" ]
+then
+number_of_operations=-1
+else
+number_of_operations=$ctl_number_of_operations
+fi
+
+if [ -z "$ctl_max_running_time_duration" ]
+then
+max_running_time_duration=PT10M
```

Comment:
I notice this is 10 min and got copied from sdk run file, however in sdk we give explicit option ctl_max_running_time_duration="PT24H" on ctl run, so either make sure we pass this in LI ctl as well , or make this default as PT24H here itself.